Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/shopify/use-user-order.md

# useUserOrder composable

> `useUserOrder` composition API function is responsible, as its name suggests, for interactions with the user's order history from your eCommerce. This function returns the following values:

Reviewer: ```suggestion
`useUserOrder` composition API function allows interactions with the customer's order history. This function returns the following values:
```
